front/pages/api/w/[wId]/data_sources/managed.ts

```diff
@@ -428,6 +430,10 @@ async function handler(
         connectorId: connectorsRes.value.id,
         connectorProvider: provider,
       });
+      const dataSourceType = await getDataSource(auth, dataSource.name);
```

Comment:
Same here why not using `dataSource` from above?

Reply:
Same here.